docs/guides/vault_secrets_manager.md

```diff
@@ -87,12 +74,12 @@ To overcome this limitation, you can integrate the HashiCorp Vault provider with
       delete_all_versions = true
       data_json = jsonencode(
         {
-          dns_zone_name = stackit_dns_zone.example.dns_name,
+         grafana_password = stackit_argus_instance.example.grafana_initial_admin_password,
         }
       )
     }
     ```
 
 ### Note
 
-This example can be adapted for various resources within the provider by replacing the DNS zone with the appropriate resource and associated variables.
+This example can be adapted for various resources within the provider by replacing the Argus Monitoring Grafana password with the appropriate resource and associated variables.
```

Comment:
For various resources or any other secret that the user wants to set into the Secrets Manager instance